src/pages/Thanks/ThanksPage.vue

```diff
@@ -300,10 +268,6 @@ export default {
 		const hasAutoDeposit = data?.my?.autoDeposit?.id ?? false;
 		this.isAutoDepositSubscriber = !!(hasAutoDeposit && !this.isMonthlyGoodSubscriber);
 
-		const isLoggedIn = data?.my?.userAccountId?.id !== null;
```

Comment:
Same with this props, maybe there are fields we can remove from the `thanksPageQuery`, which could speed up the initial load of the page.